tasks/update-granules-cmr-metadata-file-links/index.js

```diff
@@ -56,6 +59,27 @@ async function updateEachCmrFileAccessURLs(
   }));
 }
 
+async function updateCmrFileInfo(cmrFiles, granulesByGranuleId) {
+  const updatedGranulesByGranuleId = cloneDeep(granulesByGranuleId);
+  const promises = cmrFiles.map(async (cmrFileObject) => {
+    const cmrFile = await updatedGranulesByGranuleId[cmrFileObject.granuleId].files.find(isCMRFile);
```

Comment:
is there value to checking for unidentified/not present in any of these? I can see how if cmrFileObject.granuleId isn't *in* updatedGranulesByGranuleId we'd want an error thrown because something has gone wrong, and it looks like files is [] at bare minimum, maybe we can be certain these are all defined

Reply:
awesome, descriptive errors in case feel much more future proof. I do think proper typing would be the ideal but I feel the need to keep this moving